src/haxelib/SemVer.hx

```diff
@@ -97,6 +97,10 @@ abstract SemVer(String) to String {
 	inline function get_valid()
 		return isValid(this);
 
+	@:op(a > b) static inline function greaterThan(a:SemVer, b:SemVer) {
```

Comment:
It seems reasonable to have comparison operators for SemVer, though I usually follow "refactor is used more than twice" rule :) I don't have a strong opinion here...